phases/codesign.go

```diff
@@ -84,6 +84,11 @@ func AutoCodesign(bitriseYML bitriseModels.BitriseDataModel, searchDir string) (
 		if uploadIOS == answerYes {
 			log.Debugf("Exporting iOS codesigning files.")
 
+			fmt.Println()
```

Comment:
`To upload iOS code signing files...` Think this is not really relevant here as codesigning files are being upooaded, and users will miss it in the log, or get confused.